kuma/core/middleware.py

```diff
@@ -14,6 +15,11 @@
 from .views import handler403
 
 
+SITE_HOST = urlparse(settings.SITE_URL).netloc
+CDN_HOST = urlparse(settings.STATIC_URL).netloc
+USING_CDN = CDN_HOST and (CDN_HOST != SITE_HOST)
```

Comment:
I'm not certain about this approach of extracting a "valid" `CDN_HOST`. I like that it's "automatic", in that it requires no new environment variables while at the same time capturing (I think) our actual practice of placing the CDN host into the `STATIC_URL`. On the other hand, it's less configurable that simply adding another variable, like `CDN_HOST`, that can be set from the environment. @jwhitlock What are your thoughts?

Reply:
I think it needs to be a new setting. On SCL3, ``STATIC_URL`` is ``'https://developer.cdn.mozilla.net/static/'``, but that's the user-facing URL. I think the CDN calls ``'https://developer-origin.cdn.mozilla.net/static/'`` when it needs to request a file it hasn't seen before. I'm not sure what the incoming domain will be in AWS - we may need a list of CDN hosts...

Reply:
I see that I was "having it both ways", knowing that `developer.cdn.mozilla.net` was the user-facing URL, and at the same time thinking it was how the CDN would make requests to MDN. Ugh, that's embarrassing. Yes, we'll need a list of CDN hosts, thanks @jwhitlock.